dev_guide/deployments.adoc

```diff
@@ -109,6 +109,30 @@ created each time a new version of the `origin-ruby-sample:latest` image reposit
 <5> The link:#rolling-strategy[rolling strategy] is the default and may be omitted.
 ====
 
+== Deployment Client Tool Operations
+Use the 'ocs deploy' command with available options to start, view, cancel, or
+retry a deployment.
+
+----
+$ oc deploy <deployment_config> [--options]
+----
+
+[cols= 2]
+|===
+|`oc deploy <deployment_config> --latest`
+|Starts the specified deployment
+
+|`oc deploy <deployment_config>`
+|Dsiplays the specified deployment
+
+|`oc deploy <deployment_config> --retry`
+|Attempts to retry to deploy the last deployment. The deployer pod and any hook pods are deleted.
+
+|`oc deploy <deployment_config> --cancel`
+|Cancels the deployment that is currently in progress
```

Comment:
@bfallonf @ironcladlou I think longer-form explanations about CLI tasks, why you'd want to use them, any actual examples, etc, should stick to their respective dedicated topic (in this case: `dev_guide/deployments`), while the `basic_cli_operations` sections should stick to comprehensive but bare-bones tables (in this case., all deployment cmds) that act as a quick reference. And in those respective `basic_cli_operations` sections, link back to their dedicated topics for more. E.g., at the start-ish of the "Deployment Operations" section, link to `dev_guide/deployments`. Or, in a command's table entry, link to the `dev_guide/deployments.html#appropriate-anchor`.

Most of the other sections in  `basic_cli_operations` kinda already do this (altho they still need cross-links). It's mainly just the "Deployment Operations" section that was a bit of an outlier, what with all that detailed info on rollbacks and canceling. The "Deployment Rollbacks" and "Canceling a Deployment" sections seem like they should go into `dev_guide/deployments`, but mention their cmds quickly in a `basic_cli_operations` table.